app/(frontend)/(client-side)/product/[data]/page.tsx

```diff
@@ -0,0 +1,60 @@
+'use client'
+
+import React, { useState } from 'react';
+import Image from 'next/image';
+import Button from "@/app/components/ui/Button";
+import ColorDisplay from "@/app/components/admin/ColorDisplay";
+
+const ProductDetailPage = ({ params }: { params: { data: string } }) => {
+
+  const [idSelected, setId] = useState<number>(0)
+  const [sizeSelected, setSizeId] = useState<number>()
+  const productInfo = JSON.parse(decodeURIComponent(params.data))
+
+  return (
+    <form className='grid grid-cols-1 md:grid-cols-2 p-4 md:p-8'>
+      <section className='flex justify-center p-4 h-full w-full'>
+        <Image alt={productInfo.name} src={productInfo.Attribute[idSelected? idSelected: 0].image} width={450} height={450} className='max-h-full w-auto' />
+      </section>
+      <section className='font-normal p-4'>
+        <header className='font-semibold text-xl'>{productInfo.name}</header>
+        <p>{productInfo.price} BOB</p>
+        <section className='my-3'>
```

Comment:
What does the my-3 class mean?

Reply:
hmmm is bootstrap?

Reply:
the m in my-3 stands for margin, and y stands for the vertical direction. The number 3 indicates the size of the margin.